easybuild/easyconfigs/p/pytest/pytest-3.8.0-foss-2018a-Python-3.6.4.eb

```diff
@@ -11,11 +11,7 @@ toolchain = {'name': 'foss', 'version': '2018a'}
 
 dependencies = [('Python', '3.6.4')]
 
-exts_defaultclass = 'PythonPackage'
-exts_default_options = {
-    'download_dep_fail': True,
-    'use_pip': True,
-}
+use_pip = True
```

Comment:
@SimonPinches the failing tests say that you should add this line as well (this is fairly new):
```
sanity_pip_check = True
```

Reply:
@smoors, apologies for the delay in commenting, but this is added now.